Rakefile

```diff
@@ -7,7 +8,7 @@ task default: %w(lint test)
 
 # Formatters
 desc 'Run formatters'
-task format: %w(format:cucumber)
+task format: %w(rubocop:auto_correct format:cucumber)
```

Comment:
This would cause triple-nested rake tasks `lint:auto_correct:ruby` and `lint:auto_correct:cucumber`. I'd prefer only 2 nesting levels here: `lint` for linters and `format` for code formatters, i.e. as is. What do you think?